src/assets/translations/en/main.json

```diff
@@ -1880,7 +1880,8 @@
   },
   "ledgerOpenApp": {
     "title": "Open the %{appName} App",
-    "description": "To continue, you will need to open the %{appName} app on your Ledger device."
+    "description": "To continue, you will need to open the %{appName} app on your Ledger device.",
+    "signingDescription": "Your transaction will require signing on-device after you open the app"
```

Comment:
Summoning @twblack88 here but feels like this should be more along the lines of "Signing prompt will automatically pop-up in your device after opening the app"

Reply:
Beautiful. Happy with this if @twblack88 is.